src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -169,7 +169,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
         // Triggers the saveAddressBook method by executing an add command
         String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
                 + EMAIL_DESC_AMY + ROOMNUMBER_DESC_AMY + TELEGRAM_DESC_AMY + BIRTHDAY_DESC_AMY;
-        Person expectedPerson = new PersonBuilder(AMY).build();
+        Person expectedPerson = new PersonBuilder(AMY).withFreeTimeTags().build();
```

Comment:
May I know why is this change needed? Looks like the constructor of PersonBuilder with the parameter AMY will already initialize the freeTimeTags?

Reply:
I added it while debugging as without it, it will cause the following test cases to fail as the expected/actual models do not match:
execute_storageThrowsIoException_throwsCommandException
execute_storageThrowsAdException_throwsCommandException

Reply:
Hmmm I thought `new PersonBuilder(AMY)` and `new PersonBuilder(AMY).withFreeTimeTags()` should behave the same? Or what are their difference?